hack/generate-operator-bundle.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+#
+# Generate an operator bundle for publishing to OLM. Copies appropriate files
+# into a directory, and composes the ClusterServiceVersion which needs bits and
+# pieces of our rbac and deployment files.
+#
+# Usage ./hack/generate-operator-bundle.py OUTPUT_DIR PREVIOUS_VERSION GIT_NUM_COMMITS GIT_HASH HIVE_IMAGE
+#
+# Commit count can be obtained with: git rev-list 9c56c62c6d0180c27e1cc9cf195f4bbfd7a617dd..HEAD --count
+# This is the first hive commit, if we tag a release we can then switch to using that tag and bump the base version.
+
+import datetime
+import os
+import sys
+import yaml
+import shutil
+
+# This script will append the current number of commits given as an arg
+# (presumably since some past base tag), and the git hash arg for a final
+# version like: 0.1.189-3f73a592
+VERSION_BASE = "0.2"
```

Comment:
Maybe we can start at 0.1

Reply:
Ahhh!! I swear I had changed this. Wonder why it was not changed.. Thanks to catch it though.. Will fix this.